src/AboutContent.js

```diff
@@ -9,9 +9,36 @@ const AboutContent = function ( props ) {
     <wrapper>
       
       <Header as='h1' style={{ fontSize: '4em' }}>About the Cliff Effects Tool</Header>
-      <p style={{ fontSize: '1.33em' }}>
-        Fill in information about the tool here.
-      </p>
+
+      <Header as='h3'>What is this?</Header>
+      <p>This tool can help show how a change in income affects how much someone receives in public assistance from SNAP and Section 8 Housing Voucher benefits. It was designed for the case managers at <a href='http://www.prohope.org/'>Project Hope</a> with the aim of helping to predict changes in their client's benefits.</p>
+
+      <Header as='h3'>Why are you doing this?</Header>
+      <p>When a slight change in a household’s circumstances - say, a slight pay raise - disproportionately lowers their benefits, that’s called a “cliff effect”. The household is working to increase what they earn, but they end up with a net loss that actually puts them further behind. These “cliff effects” prevent many families from actually getting off of public assistance programs.</p>
+      <p>Cliff effects are also very hard to predict. The interactions between income, household size, many other criteria, and the effects of the programs themselves impact each other in unexpected ways. We're exploring ways to deal with this issue of complexity and help families better understand and predict their situation.</p>
+      <p><a href='#learnMore'>Learn more about cliff effects</a></p>
```

Comment:
This link is currently dead - it reroutes to /#/learnMore, which is a non-existent page (that does not 404 for ... I don't know why, and we should fix that separately). Either fix (to route to the header below) or remove for now as we don't introduce any broken links, especially as we close in on a deadline and have other priorities.